app/packs/components/PackContents/PackContents.tsx

```diff
@@ -1,25 +1,49 @@
-import { Button, Card } from "@components"
+import { Button } from "@components"
 import { ICard } from "@interfaces"
 import { useUserStore } from "@stores"
 
 import "./packContents.scss"
+import { useEffect, useState } from "react"
+import PackCarousel from "./components/PackCarousel/PackCarousel"
+import MultiCardShow from "./components/MultiCardShow/MultiCardShow"
```

Comment:
To keep it consistent with the project, lets have an `index.ts` file inside the PackContents/components folder i.e.:

[UserPacks/components/index.ts](https://github.com/dustinstacy/nexus-dawn/blob/main/app/packs/components/UserPacks/components/index.ts)

```solidity
export { default as Carousel } from './Carousel/Carousel'
export { default as UserPack } from './UserPack/UserPack'
```

so that our import can be:

```solidity
    import {MultiCardShow} from "./components"
```